src/_js/main.mjs

```diff
@@ -48,11 +48,36 @@ const updateTwitterTimeline = async () => {
   }
 };
 
+const objects = document.querySelectorAll('object');
+const updateSvgs = async () => {
+  // For any object containing an SVG, toggle dark mode on the SVG.
+  objects.forEach(async (object)=> {
+    const contentDocument = object.contentDocument;
+    if (!contentDocument) return;
+    // Make sure the document is ready
+    if (contentDocument.readyState == "loading") {
+      await new Promise((resolve, reject)=>{
+        contentDocument.addEventListener("readystatechange", (e) => {
+          if (contentDocument.readyState != "loading") {
+            resolve();
+          }
+        })
+      });
+    }
+    const rootElement = contentDocument.rootElement;
+    if (!rootElement) return;
+    if (rootElement.tagName == "svg") {
+      rootElement.classList.toggle('dark', darkModeToggle.mode === 'dark');
+    }
+  });
+}
+
```

Comment:
Let's inline this `<script>` in the blog post itself, since that's the only page that needs it.

Reply:
We found a better way offline (inlining the SVG, which removes the need for any JS at all, and also lets us use CSS variables).